examples/command.rs

```diff
@@ -0,0 +1,433 @@
+#![allow(clippy::type_complexity)]
+
+use std::ops::DerefMut;
+
+use parsers::vec2::Vec2 as Vec2Parser;
+use parsers::vec3::Vec3 as Vec3Parser;
+use valence::prelude::*;
+use valence_command::graph::CommandGraphBuilder;
+use valence_command::handler::CommandResultEvent;
+use valence_command::parsers::entity_selector::{EntitySelector, EntitySelectors};
+use valence_command::parsers::strings::{GreedyString, QuotableString};
+use valence_command::parsers::CommandArg;
+use valence_command::scopes::CommandScopes;
+use valence_command::{parsers, Command, CommandApp, CommandScopeRegistry, ModifierValue};
+use valence_command_derive::Command;
+use valence_server::op_level::OpLevel;
+
+const SPAWN_Y: i32 = 64;
+
+#[derive(Command, Debug, Clone)]
+#[paths("teleport", "tp")]
+#[scopes("valence:command:teleport")]
```

Comment:
I think we should use `.` as the scope separator instead of `:`
- More familiar to users of luckperms
- Avoids confusion with namespaced identifiers.